scripts/inner/kubestellar

```diff
@@ -88,10 +88,10 @@ function ensure_imws() {
             imw=${i#"root:"}
```

Comment:
This should be unnecessary now, right? I would prefer to remove it and find errors in callers.